File: sobabank_back/src/com/bl/model/Demande.java
package com.bl.model;

import java.util.Date;

public class Demande {
	private Long id;
	private Date dateDemande;
	private String libelle;
	private String status;
	private Boolean isValid;

	public Demande(){
		
	}
	
	public Demande(Long id, Date dateDemande, String libelle, String status, Boolean isValid) {
		super();
		this.id = id;
		this.dateDemande = dateDemande;
		this.libelle = libelle;
		this.status = status;
		this.isValid = isValid;
	}

	public Long getId() {
		return id;
	}
	
	public void setId(Long id) {
		this.id = id;
	}
	
	public String getLibelle() {
		return libelle;
	}
	
	public void setLibelle(String libelle) {
		this.libelle = libelle;
	}
	
	
	public String getStatus() {
		return status;
	}
	
	public void setStatus(String status) {
		this.status = status;
	}
	
	public Boolean getIsValid() {
		return isValid;
	}
	
	public void setIsValid(Boolean isValid) {
		this.isValid = isValid;
	}

	public Date getDateDemande() {
		return dateDemande;
	}

	public void setDateDemande(Date dateDemande) {
		this.dateDemande = dateDemande;
	}
	
	
	
	
}

File: SobaBank/src/app/vue_admin/test_class/demande.ts
export class Demande{
	id: string;
	agent: string;
	status: string;
	dateDem: Date;

	constructor(id: string,agent: string,status: string, dateDem: Date){
		this.id = id;
		this.agent = agent;
		this.status = status;
		this.dateDem = dateDem;
	}

	getId(){
		return this.id;
	}

	getAgent(){
		return this.agent;
	}

	getStatus(){
		return this.status;
	}

	getDateDem(){
		return this.dateDem;
	}
}